.circleci/config.yml

```diff
@@ -0,0 +1,133 @@
+    # Dependencies
+    #   This would typically go in either a build or a build-and-test job when using workflows
+    # Restore the dependency cache
+    - restore_cache:
+        keys:
+        # This branch if available
+        - v1-dep-{{ .Branch }}-
+        # Default branch if not
+        - v1-dep-master-
+        # Any branch if there are none on the default branch - this should be unnecessary if you have your default branch configured correctly
+        - v1-dep-
+    # This is based on your 1.0 configuration file or project settings
+    - run: sudo chmod o+w /opt
+    - run: pip install --upgrade pip
+    - run: pip install --upgrade httplib2 jira oauth2client parallel pylint
+    - run: pip install -I alembic==0.9.1
+    # Save dependency cache
+#    - save_cache:
+#        key: v1-dep-{{ .Branch }}-{{ epoch }}
+#        paths:
+#        # This is a broad list of cache paths to include many possible development environments
+#        # You can probably delete some of these entries
+#        - vendor/bundle
+#        - ~/virtualenvs
+#        - ~/.m2
+#        - ~/.ivy2
+#        - ~/.bundle
+#        - ~/.go_workspace
+#        - ~/.gradle
+#        - ~/.cache/bower
+    # Test
+    #   This would typically be a build job when using workflows, possibly combined with build
+    # This is based on your 1.0 configuration file or project settings
+    - run: ./ci/test.sh
+    # Deployment
+    # Your existing circle.yml file contains deployment steps.
+    # The config translation tool does not support translating deployment steps
+    # since deployment in CircleCI 2.0 are better handled through workflows.
+    # See the documentation for more information https://circleci.com/docs/2.0/workflows/
+    # Teardown
+    #   If you break your build into multiple jobs with workflows, you will probably want to do the parts of this that are relevant in each
+    # Save test results
+    - store_test_results:
+        path: /tmp/circleci-test-results
+    # Save artifacts
+    - store_artifacts:
+        path: /tmp/circleci-artifacts
+    - store_artifacts:
+        path: /tmp/circleci-test-results
+        
+    # Deploy master to the pmi-drc-api-test project whenever the tests pass
+    - run: ./ci/activate_creds.sh ~/gcloud-credentials.key
+    - run:
+        command: ./tools/upgrade_database.sh -i https://pmi-drc-api-test.appspot.com --creds_file ~/gcloud-credentials.key
+        working_directory: ~/all-of-us/raw-data-repository/rest-api
+
+    - run: ./ci/deploy.sh pmi-drc-api-test config/config_test.json ~/gcloud-credentials.key
+    - run: ./rest-api/test/test_server.sh -i https://pmi-drc-api-test.appspot.com -c ~/gcloud-credentials.key
+
+    # To deploy to staging, cut a GitHub release on a green build, with tag of
+    # the form v0-1-1-rc0. We use tags as AppEngine version names, so they must
+    # follow AE rules (no dots).
+#    steps:
+#      # This should match _RELEASE_TAG_RE in release_notes.py.
+  deploy_staging:
```

Comment:
Make sure this runs when there's a tag matching the pattern below, and doesn't run otherwise.

Reply:
done